tests/Behat/Element/Element.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\AdminOrderCreationPlugin\Behat\Element;
+
+use Behat\Mink\Driver\DriverInterface;
+use Behat\Mink\Element\DocumentElement;
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Behat\Mink\Selector\SelectorsHandler;
+use Behat\Mink\Session;
+
+/**
+ * It is a copy-passed class from UrbanaraCatalogPromotionBundle
```

Comment:
I think that if we copy this class we can also improve it, and avoid this comment 😄